### PR TITLE
Try to fix boskos config update job

### PR DESCRIPTION
**Pull request by krzyzacy**
Or is there other workarounds?

/assign @fejta

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -9940,7 +9940,7 @@
   },
   "maintenance-boskos-config-upload": {
     "args": [
-      "cd boskos && make update-config"
```
Comment (ixdy) [inline jobs/config.json]: maybe instead try
```json
"args": [
  "make", "-C", "boskos", "update-config"
]
```
?

Comment (krzyzacy) [reply jobs/config.json]: done

File: jobs/config.json [hunk 2]
```diff
@@ -9940,7 +9940,11 @@
...
+      "make",
+      "--",
```
Comment (ixdy) [inline jobs/config.json]: you don't want the `--` before `-C`, as that will make `make` not treat `-C` as a flag to `make`.

Comment (ixdy) [reply jobs/config.json]: oh, maybe you need it so execute.py doesn't interpret the `-C`?

this is confusing.

**Comment by ixdy** [comment]
/lgtm

I think

File: jobs/config.json (hunk 2 above)
Comment (fejta) [reply jobs/config.json]: I would expect it to work with or without the --

**Comment by krzyzacy** [comment]
(I'll try without `--`)

**Pull request merged**